src/PlaywrightRunner.ts

```diff
@@ -33,11 +33,14 @@ const getBrowserTest = (
       ...test.context,
       config: {
         ...test.context.config,
+        // @ts-ignore
         browserName: browser,
         device,
         displayName: {
           name: displayName
-            ? `${playwrightDisplayName} ${displayName.name}`
+            ? `${playwrightDisplayName} ${
```

Comment:
Maybe `typeof displayName === 'string' ? displayName : displayName.name` is better?